src/solution.jl

```diff
@@ -13,10 +14,21 @@ mutable struct Solution
         # M3_ij is the number of subsequences in which the number of
         # cars that require oj is greater than or equal to p(oj)
 
-    Solution(nC,nO) = new(
-        zeros(Int,nC),
+    Solution(nC::Int,nO::Int) = new(
+        collect(1:nC),
         zeros(Int,nO,nC),
         zeros(Int,nO,nC),
         zeros(Int,nO,nC)
     )
 end
+
+# Build an initial
+function init_solution(nom_fichier::String, type_fichier::String)
+    # Read in data files
+    instance = parser(nom_fichier, type_fichier)
+
+    n = length(instance.color_code)
+    m = length(instance.HPRC) + length(instance.LPRC) # number of ratio
```

Comment:
Why not  `m = instance.nb_HPRC + instance.nb_LPRC` ?

Reply:
In fact it would be better, I will change that.